### PR TITLE
Allow "dibs" to claim a post

### PR DESCRIPTION
**Pull request by DavidArchibald**
Just for fun add dibs as a way to claim a post.

### CODE REVIEW

File: tor/core/inbox.py [hunk 1]
```diff
@@ -93,7 +93,8 @@ def process_reply(reply, config):
 
         if (
             'done' in r_body or
-            'deno' in r_body  # we <3 u/Lornescri
+            'deno' in r_body or # we <3 u/Lornescri
+            'dibs' in r_body
```
Comment (itsthejoker) [inline tor/core/inbox.py]: Putting it here would only allow it to finish a post, not claim one.

Comment (DavidArchibald) [reply tor/core/inbox.py]: Oops it started with "d" and fit the pattern so I just forgot basic logic 😅

File: tor/core/inbox.py [hunk 2]
```diff
@@ -86,14 +86,17 @@ def process_reply(reply, config):
...
+        ):
             process_claim(reply, config)
             reply.mark_read()
             return
 
         if (
             'done' in r_body or
-            'deno' in r_body  # we <3 u/Lornescri
+            'deno' in r_body # we <3 u/Lornescri
```
[the remark this answers is not in the document]
Comment (DavidArchibald) [reply tor/core/inbox.py]: Maybe I shouldn't edit on my phone... I'm so used to black doing it for me, sorry for making this much harder than it has to be.

**Comment by PejmanPoh** [comment]
👍

**Pull request closed without merging**